setup.py

```diff
@@ -45,26 +44,30 @@ def run_tests(self):
 
 class BuildDocsCommand(setuptools.command.build_py.build_py):
     apidoc_command = (
-        'sphinx-apidoc', '-f', '-o', 'docs', '--no-toc', 'coalib'
+        'sphinx-apidoc', '-f', '-o', 'docs',
+        '--no-toc',
```

Comment:
I don't really get this reformatting. It is not consistent in itself. Lines neither hold the same number of elements nor are they the same length.
  Or is it grouping by type of parameter?

Reply:
This is being formatted by moban, and the logic is re-used between core and -bears.
See https://github.com/coala/coala-bears/blob/5ca3eb7/setup.py#L28 where bears has a different set of parameters for this.